articles/container-apps/java-metrics.md

```diff
@@ -0,0 +1,94 @@
+---
+title: How to enable Java metrics for Java apps in Azure Container Apps
+description: Java metrics and configuration Azure Container Apps.
+services: container-apps
+author: 
+ms.service: container-apps
+ms.custom: 
+ms.topic: 
+ms.date: 
+ms.author: 
+---
+
+# Java metrics for Java apps in Azure Container Apps
+
+JVM (Java Virtual Machine) metrics are critical for monitoring the health and performance of Java applications.
+They provide insights into various aspects of the JVM, such as memory usage, garbage collection, thread count, and more.
+
+## Azure Container Apps collect list of Java metrics
+
```

Comment:
Please consider if it is helpful to put this link on our doc, for customers to have more information if they need to. https://opentelemetry.io/docs/specs/semconv/runtime/jvm-metrics/ But if it is too complicated to explain how our metrics map to the OTel standard, we can ignore the link.